### DISCUSSION TITLE
How can I turn on encryption on a long with `nginx` webserver?

### DISCUSSION THREAD

**Discussion by quyleanh** [AdguardTeam/AdGuardHome#2782] [Q&A]
I install Adguard Home along with LEMP server. The port `443` is already use for `nginx` server. I want to make Adguard Home's admin interface site available with SSL encryption.

How can I achieve it? Thank you.

**Comment by ameshkov** [comment]
Here's what you should do:

1. You should configure nginx to do the encryption
2. Use `proxy_pass` to pass the traffic to AdGuard Home, something like this: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#reverseproxy

**Reply by quyleanh** [reply]
Thank you for your support. In my case, the URL is `http://aghome.YOUR_SERVER/`, since my  `http://YOUR_SERVER/` URL has different IP address with  `http://aghome.YOUR_SERVER/`.

**Reply by ameshkov** [reply]
Well, that's not a problem, a single `proxy_pass` would be enugh then, you won't need `proxy_redirect ` and `proxy_cookie_path `
